### DISCUSSION TITLE
How to report a Wine bug if the problem occurs in Wineskin?

### DISCUSSION THREAD

**Discussion by emendelson** [The-Wineskin-Project/WineskinServer#199] [General]
I was able to making a working wrapper for WordPerfect 7 for Windows by following the advice on this page:

https://appdb.winehq.org/objectManager.php?sClass=version&iId=17581&iTestingId=66182

It mentions a bug in Wine that doesn't create directories when WordPerfect setup is run, and gives a workaround (which works). The bug has now been there for many, many years, and it seems worth fixing. Would I have to install Wine and attempt a Wine setup before i can file a bug report at WineHQ?

I hope this is a suitable question for this page.

**Comment by Gcenx** [comment]
To file a valid bug report to upstream wine would require using a none CX engine.

When using a none CX engine use `Winehq Engine Launch` this will open a terminal windows with `$PATH` & all other environment settings configured to run the Engine like upstream wine.

If you require a more recent version of wine those can now be installed via brew, so that’s wine-6.0.2/6.23 packages (haven’t pushed 7.0-rc builds but those can be build via Macports-wine)

**Reply by emendelson** [reply]
OK - thank you for explaining that (I would never have figured it out on my own). I'll experiment with this - and, before I file a bug report, I'll try out the more recent builds. Meanwhile, if anyone knows the answer to the issue I posted (how to get the path of a dropped file into a variable in the startup script), that would be terrific. Thanks again!
